### PR TITLE
Clarify alias usage

### PR DESCRIPTION
**Pull request by andrewyatz**
See #329

We have removed the reference to aliases. Instead we have formalised alternative checksum support. Alternative checksums must not clash with any other identifier space.

### CODE REVIEW

File: refget.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ Content-type: text/vnd.ga4gh.refget.v1.0.0+plain
 
 | Parameter | Data Type | Required | Description                                                                                                                                                                                                         |
 |-----------|-----------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `id`      | string    | Yes      | A string specifying the sequence to be returned. The identifier shall be a checksum derived from the sequence using one of the supported checksum algorithms, or an alias for the sequence supported by the server. |
+| `id`      | string    | Yes      | A string specifying an identifier for the sequence to be returned. The identifier shall be a checksum derived from the sequence using one of the supported checksum algorithms, or a server specific checksum algorithm identifier. |
```
Comment (jmarshall) [inline refget.md]: Sounds a bit like `id` could be `FREDS_ALGORITHM` — i.e. an identifier for the algorithm. Perhaps something like _…using one of the defined checksum algorithms or a server-specific checksum algorithm…_?

File: refget.md [hunk 2]
```diff
@@ -167,7 +167,7 @@ Content-type: application/vnd.ga4gh.refget.v1.0.0+json
 
 | Parameter | Data Type | Required | Description                                                                                                                                                                                                         |
 |-----------|-----------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `id`      | string    | Yes      | A string specifying identifier to retrieve aliases for. The identifier shall be a checksum derived from the sequence using one of the supported checksum algorithms, or an alias for the sequence supported by the server. |
+| `id`      | string    | Yes      | A string specifying an identifier to retrieve metadata for. The identifier shall be a checksum derived from the sequence using one of the supported checksum algorithms, or a server specific checksum algorithm identifier. |
```
Comment (jmarshall) [inline refget.md]: (ditto)

File: refget.md [hunk 3]
```diff
@@ -356,6 +356,17 @@ GAGACTGCTG
 
 Any bytes added for formatting to the returned output should not be taken in to account when processing a Range request.
 
+## Alternative Checksum Algorithms
+
+Refget implementations MUST support the `MD5` identifier space SHOULD support `TRUNC512`. Non-standard identifiers are allowed but MUST conform to the following requirements:
+
+1. Non-standard identifiers must be based on a hash algorithm dervied from sequence content
```
Comment (jmarshall) [inline refget.md]: typo _derived_

File: refget.md [hunk 4]
```diff
@@ -356,6 +356,17 @@ GAGACTGCTG
 
 Any bytes added for formatting to the returned output should not be taken in to account when processing a Range request.
 
+## Alternative Checksum Algorithms
+
+Refget implementations MUST support the `MD5` identifier space SHOULD support `TRUNC512`. Non-standard identifiers are allowed but MUST conform to the following requirements:
```
Comment (jmarshall) [inline refget.md]: Something missing in this sentence…

**Comment by andrewyatz** [comment]
Okay the changes you suggested have been made. Cheers!

**Pull request merged**